packages/app/src/containers/SignupPage/index.tsx

```diff
@@ -0,0 +1,107 @@
+import * as React from 'react';
+import {
+    ImageBackground,
+    Image,
+    Text,
+    View,
+    StyleSheet
+} from 'react-native';
+import { Button, Input } from "react-native-elements";
+
+interface SignupPageState {
+    fullname: string;
+    email: string;
+    password: string;
+}
+
+interface OwnProps {
+    navigation: any
+}
+
+type Props = OwnProps;
+
+class SignupPage extends React.Component<Props, SignupPageState> {
+    constructor(props: Props) {
+        super(props);
+
+        this.state = {
+            fullname: '',
+            email: '',
+            password: '',
+        };
+    }
+
+
+    public componentDidMount() {
+    }
+
+    public render() {
+
+        return (
+            <React.Fragment>
+                <ImageBackground source={require("../../../assets/images/bg.jpg")} resizeMode="cover" style={styles.container}>
```

Comment:
this path doesnt exist in the repo, image is not found

Reply:
@petermsouzajr Images folder did not push to the repo because of gitignore. Now I have renamed the folder and it is fixed now